packages/react-components/react-charts-preview/src/utilities/utilities.ts

```diff
@@ -189,6 +197,14 @@ function multiFormat(date: Date, locale?: d3TimeLocaleObject) {
   const formatMonth = timeFormat('%B');
   const formatYear = timeFormat('%Y');
 
+  const timeSecond = useUTC ? d3UtcSecond : d3TimeSecond;
+  const timeMinute = useUTC ? d3UtcMinute : d3TimeMinute;
+  const timeHour = useUTC ? d3UtcHour : d3TimeHour;
+  const timeDay = useUTC ? d3UtcDay : d3TimeDay;
+  const timeMonth = useUTC ? d3UtcMonth : d3TimeMonth;
+  const timeWeek = useUTC ? d3UtcWeek : d3TimeWeek;
+  const timeYear = useUTC ? d3UtcYear : d3TimeYear;
+
   return (
     d3TimeSecond(date) < date
       ? formatMillisecond
```

Comment:
where are you using the const values defined above

Reply:
Not using currently, but this might be used in other charts in future. It is present in utilities file of V8

Reply:
I see it is being used in v8 code

Reply:
Yes, so it might be using in other charts

Reply:
This is used when a time format is provided explicitly. Applicable for all charts. See this example.
https://developer.microsoft.com/en-us/fluentui#/controls/web/linechart:~:text=LineChart%20custom%20date,Show%20code

Reply:
Ohh okay

Reply:
this comment is still active

Reply:
Did not get what to do here. These const values are defined in the utilities file. It is used by other charts

Reply:
See this PR on how to consume these 
https://github.com/microsoft/fluentui/pull/31525

Reply:
Made the change